system/ee/legacy/core/Input.php

```diff
@@ -988,6 +988,8 @@ public function _clean_input_data($str)
             return $new_array;
         }
 
+        $str = (string) $str;
```

Comment:
Should we check if it's NULL or number instead, and if yes just return here and stop following execution? Similar to what you did in the other file here